lldb/source/Plugins/TypeSystem/Swift/TypeSystemSwiftTypeRef.cpp

```diff
@@ -1587,7 +1643,64 @@ TypeSystemSwiftTypeRef::GetPointerType(opaque_compiler_type_t type) {
 llvm::Optional<uint64_t>
 TypeSystemSwiftTypeRef::GetBitSize(opaque_compiler_type_t type,
                                    ExecutionContextScope *exe_scope) {
-  return m_swift_ast_context->GetBitSize(ReconstructType(type), exe_scope);
+  auto impl = [&]() -> llvm::Optional<uint64_t> {
+    // Bug-for-bug compatibility. See comment in SwiftASTContext::GetBitSize().
```

Comment:
How come this implementation is so much different from `master-next`? https://github.com/apple/llvm-project/commit/dab2fc65fba8f735be7f315c24c587c20525a412#diff-4f052b540e66838ef123c3a839bacf1cR1617-R1629

Reply:
For example `GetAsClangTypeOrNull` exists in this PR but not in the `master-next` version?

Reply:
Thank you for catching this! I mush have cherry-picked an older version of the commit. Wow.

Reply:
Hmm.. no. git diff origin/swift/master doesn't show a difference. Is it possible that this is the state of one of the earlier commits in the PR and not the final one?

Reply:
the reason I noticed this is because when I cherry picked #1779, there were changes to `GetAsClangTypeOrNull` that did not exist at all on `swift/master-next`.

Reply:
I am confused 🤔

Reply:
It seems I have done something wrong in my cherry-pick in #1779.